src/libstd/process.rs

```diff
@@ -134,6 +134,16 @@ use crate::sys_common::{AsInner, AsInnerMut, FromInner, IntoInner};
 /// the parent process wait until the child has actually exited before
 /// continuing.
 ///
+/// # Warning
+///
+/// On some system, calling [`wait`] or similar is necessary for the OS to release resources. A
+/// process that terminated but has not been waited on is still around as a „zombie“. Leaving too
```

Comment:
This word `„zombie“` is weird.

Reply:
Are you referring to the word itself (zombie is actually the official unix terminology for that state), or to the fact it is in quotes to denote it is a name as opposed to just a metaphor? Maybe I should not have used the quotes as they are in Czech but prefer either plain ASCII ones or correct English quotation (“…”).

Reply:
Personally I prefer ASCII ones.